website/content/api-docs/json-jobs.mdx

```diff
@@ -1055,11 +1055,23 @@ README][ct].
   - `"noop"` - take no action (continue running the task)
   - `"restart"` - restart the task
   - `"signal"` - send a configurable signal to the task
+  - `"script"` - run a script
 
 - `ChangeSignal` - Specifies the signal to send to the task as a string like
   "SIGUSR1" or "SIGINT". This option is required if the `ChangeMode` is
   `signal`.
 
+- `ChangeScriptPath`  - Specifies the full path to a script that is to be
```

Comment:
Does the indentation of these need to change since it's an embedded struct?  Also, I think you DRY'd up the naming yeah? And finally, I think you need an entry for `fail_on_error` or whatever you end up naming it, yeah?

Reply:
eh. FAILURE, I forgot about json-jobs.mdx file completely. Fixed now, thanks for catching.